bazel/antlr.patch

```diff
@@ -11,4 +11,16 @@ index c6cceda13..e86533759 100755
  }
 
  ATNDeserializer::ATNDeserializer(const ATNDeserializationOptions& dso): deserializationOptions(dso) {
+diff --git a/runtime/Cpp/runtime/src/atn/LexerATNSimulator.cpp b/runtime/Cpp/runtime/src/atn/LexerATNSimulator.cpp
+index 827c3d59f..62914cf55 100755
+--- a/runtime/Cpp/runtime/src/atn/LexerATNSimulator.cpp
++++ b/runtime/Cpp/runtime/src/atn/LexerATNSimulator.cpp
+@@ -69,7 +69,7 @@ void LexerATNSimulator::copyState(LexerATNSimulator *simulator) {
+ }
+
+ size_t LexerATNSimulator::match(CharStream *input, size_t mode) {
+-  match_calls++;
++  // match_calls++;
```

Comment:
Could you open issue upstream to get this fixed, so that we won't need to carry this patch forever?

Reply:
This one is hard. 4.7.2 ANTLR is newer but has more issues (not sure why). I chose 4.7.1 since that's in google3 with similar patches. I'll send a google3 patch with this change.

Reply:
Sounds good, thanks!